packages/playwright/src/reporters/list.ts

```diff
@@ -33,9 +34,9 @@ class ListReporter extends BaseReporter {
   private _needNewLine = false;
   private _printSteps: boolean;
 
-  constructor(options: { omitFailures?: boolean, printSteps?: boolean } = {}) {
-    super(options);
```

Comment:
why is `options` not passed anymore, base.ts still reads omitFailures?

Reply:
We do not have a public `omitFailures` option, and we do not use it privately either.